schema.go

```diff
@@ -94,7 +94,7 @@ func ParseFiles(files []string) ([]*Package, error) {
 				if ok {
 					if f.TypeList {
 						switch t {
-						case "float32", "float64", "text", "binary":
+						case "float32", "float64", "text", "binary", "int32", "int64":
```

Comment:
A compiler warning would be nice here, as this feature will fail miserably for other languages than Go.

```go
case "int32", "int64":
    log.Print("colfer: WARNING: integer lists are Go only at the moment")
```

Reply:
added )